src/index.js

```diff
@@ -0,0 +1,54 @@
+import './style.css';
+import dotsImage from './img/dots.png';
+import syncImage from './img/synchronize.png';
+
+const list = [
+  {
+    description: 'Take a shower',
+    completed: '0',
+    index: '1',
+  },
+  {
+    description: 'Breakfast',
+    completed: '1',
+    index: '2',
+  },
+];
```

Comment:
- [ ] Great work on making the array of objects 👍 but it would be great if you provide boolean values to completed 🤔 since it is a requirement of your project that completed will take boolean values **(true or false)**